security/crowdsec/+PRE_DEINSTALL.pre

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# need to temporarily stop the bouncer to remove all the rules
+service crowdsec_firewall stop >/dev/null 2>&1 | :
```

Comment:
this could be the reason for "configctl crowdsec reconfigure"? it would be nicer to aim for symmetry to make this more apparent, e.g. "service crowdsec_firewall start"

Reply:
No, the reconfigure in post-install is just to make sure the scripts are called and configuration files are generated. Might be redundant. The explicit crowdsec_firewall stop is because otherwise it could recreate the rules as they are removed.